[KB-251] Quantile Transform (#161)
chhoumann/thesis-chemcam: report_thesis/src/sections/background.tex

\subsubsection{Power Transformation}

\subsubsection{Quantile Transformer}
Quantile transformation is a method that applies a non-linear transformation to map data to a uniform or normal distribution. 
This process involves mapping the data $X$ to a set of probabilities $p$ using the \gls{cdf}, which indicates the probability that a random variable will be less than or equal to a specific value in $X$'s original distribution. 

> Subsequently, the quantile function, which is the inverse of the \gls{cdf} of the desired distribution, is applied to these probabilities $p$ to generate the transformed data. 

Reviewer: ```suggestion
Subsequently, these probabilities $p$ are transformed into the desired distribution using the quantile function, which is the inverse of the \gls{cdf} of that distribution.
```

> This method forces the data to conform to the specified distribution by assuming that the original distribution has the same form\cite{Vasques2024}.

Reviewer: ```suggestion
This method forces the data to conform to the specified distribution regardless of the original distribution's form~\cite{Vasques2024}.
```
